catalog-info.yaml

```diff
@@ -84,6 +84,8 @@ spec:
     spec:
       repository: elastic/apm-agent-java
       pipeline_file: ".buildkite/load-testing.yml"
+      provider_settings:
```

Comment:
load-testing was a placeholder while migrating the jenkins specific pipeline, since it's not yet available I decided to remove the commits-based event builds